chainer/link_hooks/timer.py

```diff
@@ -20,7 +20,8 @@
 
 
 class TimerHook(link_hook.LinkHook):
-    """Link hook for measuring elapsed time of :meth:`forward`.
+    """Link hook for measuring elapsed time of
+    :meth:`Link.forward() <chainer.Link.forward>`.
```

Comment:
Summary line should not be separated.
Please join with `\` (and remove the extra spaces).